src/game/client/tf/vgui/class_loadout_panel.cpp

```diff
@@ -686,6 +686,14 @@ void CClassLoadoutPanel::OnKeyCodePressed( vgui::KeyCode code )
 			OnCommand( VarArgs("change%d", nSelected ) );
 		}
 	}
+	else if ( nButtonCode == KEY_XBUTTON_X || nButtonCode == STEAMCONTROLLER_X )
```

Comment:
Similar to the previous "enter" case, this could use a keyboard shortcut; anyone have any suggestions?